cmd/flags/flags.go

```diff
@@ -32,6 +32,15 @@ const (
 	FlagNetworkUsage = "Sets the --chain-id, --node, and --grpc-addr flags (if applicable) based on the given network moniker (e.g. local, alpha, beta, main)"
 	DefaultNetwork   = ""
 
+	FlagFaucetBaseURL      = "base-url"
+	FlagFaucetBaseURLUsage = "The base URL of the Pocket Network Faucet"
+	// TODO_UP_NEXT(@bryanchriswhite): Update to the MainNet URL once available.
+	DefaultFaucetBaseURL = "https://shannon-testnet-grove-faucet.beta.poktroll.com"
+
+	FaucetConfigPath        = "faucet-config-path"
+	FaucetConfigPathUsage   = "Path to the faucet config yaml file ($HOME/.{pocket,poktroll} and PWD are searched by default)"
```

Comment:
> HOME/.{pocket,poktroll

Can you add a TODO to revisit this?

I'd rather just fail if it's not set explicitly. Less room for error.